gamestore/src/main/java/com/company/gamestore/GamestoreApplication.java

```diff
@@ -2,8 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 
-@SpringBootApplication
+@SpringBootApplication (exclude = { SecurityAutoConfiguration.class })
```

Comment:
We could also remove the security dependency entirely since security isn't a requirement